retroshare-gui/src/gui/Posted/PostedListWidget.cpp

```diff
@@ -534,11 +630,53 @@ void PostedListWidget::applyRanking()
 		++counter;
 	}
 
+	// Card View
```

Comment:
In cardview mode, there's a variable gap between items in the list. Is it possible to remove this gap, so that all items whatever their size, get properly adjusted?